drivers/homeless_summary_report/app/views/homeless_summary_report/warehouse_reports/reports/show.haml

```diff
@@ -0,0 +1,22 @@
+:ruby
+  def details_path(*args)
+    details_homeless_summary_report_warehouse_reports_report_path(*args)
+  end
```

Comment:
It looks like this is only used once, I'd be tempted to just put the correct method inline below so we don't have embedded ruby methods in the view.  Alternatively, this could be moved to the controller.  Also, using `method_path` is probably a bad idea since it will stomp on rails magic method naming (if there were a details route within this context.

Reply:
Makes good sense.

✅